### PR TITLE
Linting fixes

### PR DESCRIPTION
**Pull request by thinkl33t**
# Description

These changes should allow the linting and formatting stages to pass.

### CODE REVIEW

**Comment by thinkl33t** [comment]
I've tested this on the simulator but not on a real badge.

Some things that need testing:

- fusb still works as expected / correctly
- [simple_tildagon guide](https://tildagon.badge.emfcamp.org/tildagon-apps/simple_tildagon/) still works correctly when followed

File: modules/fusb302b.py [hunk 1]
```diff
@@ -462,9 +462,9 @@ def get_interrupts(self):
         Interruptb = self.i2c.readfrom_mem(
             self.ADDRESS, self.good_crc_sent_int.register, 1
         )[0]
-        Interrupt = self.i2c.readfrom_mem(
-            self.ADDRESS, self.bc_level_int.register, 1
-        )[0]
+        Interrupt = self.i2c.readfrom_mem(self.ADDRESS, self.bc_level_int.register, 1)[
+            0
+        ]
```
Comment (npentrel) [inline modules/fusb302b.py:467]: I think this is the more generally accepted way of splitting this line
```suggestion
        Interrupt = self.i2c.readfrom_mem(
            self.ADDRESS, self.bc_level_int.register, 1)[0]
```

**Comment by thinkl33t** [comment]
>     * fusb still works as expected / correctly
basic testing (making sure the cable plugged into itself still glows) seems to be fine

>     * [simple_tildagon guide](https://tildagon.badge.emfcamp.org/tildagon-apps/simple_tildagon/) still works correctly when followed
all code examples within still work

**Review by npentrel — APPROVED** [review]
LGTM % one nit

File: modules/fusb302b.py (hunk 1 above)
Comment (thinkl33t) [reply modules/fusb302b.py:467]: Agreed, i tried this (and the ones above are more readable than the autogenerated one here imo) but ruff keeps reformatting it back to its preferred option - are we happy enough to leave this as is and just merge accepting it looks a little odd?

Comment (npentrel) [reply modules/fusb302b.py:467]: Funny. Sure feel free to merge

**Pull request merged**